README.md

```diff
@@ -53,17 +58,3 @@ See `CONTIBUTING.md` for information about contributing to this project.
 BSD-3
 
 See `LICENSE.txt` for details.
-
-# Related Tools
-
-* [sklearn.neural_network](http://scikit-learn.org/dev/modules/classes.html#module-sklearn.neural_network)
-* [tensorflow.contrib.learn](https://github.com/tensorflow/tensorflow/tree/r0.10/tensorflow/contrib/learn/python/learn)
-* [keras.wrappers.scikit_learn](https://keras.io/scikit-learn-api/)
-
-# Contributors
-
-* [Mike Heilman](https://github.com/mheilman/)
-* [Walt Askew](https://github.com/waltaskew/)
-* [Matt Becker](https://github.com/beckermr/)
-* [Bill Lattner](https://github.com/wlattner/)
```

Comment:
I think we should keep this or move it to a separate file [like sklearn has](https://github.com/scikit-learn/scikit-learn/blob/master/AUTHORS.rst). e.g., @wlattner didn't make a commit, but he had some useful input/feedback early on.

Reply:
Sure! I didn’t know how far to go in making it consistent with our other repos. Will do.